stockflux-watchlist/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Watchlist from './components/watchlist/Watchlist';
+import Watchlist from './components/Watchlist/Watchlist';
```

Comment:
Could we keep all folder names lowercase?

Reply:
Done